lib/rubocop/cop/base.rb

```diff
@@ -70,6 +70,7 @@ def self.documentation_url
 
       def self.inherited(subclass)
         super
+        subclass.instance_variable_set(:@gem_requirements, {})
```

Comment:
Would it be better to put `@gem_requirements ||= {}` everywhere it's referenced, instead?

Or replace the `attr_reader` with an explicit method that does this?

```diff
-attr_reader :gem_requirements
+def gem_requirements
+  @gem_requirements ||= {}
+end
```